backend/backend.go

```diff
@@ -559,8 +524,6 @@ func Initialize(ctx context.Context, config *Config) (*Backend, error) {
 			clusterVersion = status.Version
 			break
 		}
-	} else {
```

Comment:
For consistency, should we continue to report the cluster version when in dev mode? I believe there's some value in keeping it (e.g. developing checks against a dev sever).

Reply:
I had an issue with this, because dev mode uses the embedded client and not a real client, the call panics. But I can look into making it work somehow.

Reply:
Figured it out :muscle: